integration-test-remote/src/test/java/io/cdap/cdap/app/etl/google/drive/GoogleDriveTest.java

```diff
@@ -16,6 +16,7 @@
 
 package io.cdap.cdap.app.etl.google.drive;
 
+import static org.junit.Assert.assertArrayEquals;
```

Comment:
Please remove ALL static imports. Our code style does not allow that.

Reply:
done